AirCasting/Utils/StringsCustomizer.swift

```diff
@@ -0,0 +1,41 @@
+// Created by Lunar on 07/02/2022.
+//
+
+import SwiftUI
+
+final class StringCustomizer {
+    static func customizeString(_ s: String, using words: [String], fontWeight: Font.Weight = .bold, color: Color = .aircastingGray, font: Font = .body, makeNewLineAfterCustomized: Bool = false) -> Text {
+        
+        var descriptionText: Text!
+        let separatedText = s.components(separatedBy: " ")
+        var total = [String]()
+        var newL = false
+        
+        words.forEach { wd in
+            let sp = wd.components(separatedBy: " ")
+            sp.forEach({ total.append( $0 ) })
+        }
```

Comment:
this takes a while to decipher. I would suggest using [flatMap(_:)](https://developer.apple.com/documentation/swift/sequence/2905332-flatmap). It will make it a one-liner:
```Swift
let total = words.flatMap { $0.components(separatedBy: " ") }
```

But please rename this `total` variable into something more descriptive as it is saying the reader nothing ;)